Flipper/Packages/UI/Sources/Infrared/Save/InfraredSaveRemote.swift

```diff
@@ -63,13 +63,19 @@ extension InfraredView {
             Task {
                 do {
                     try await infrared.copyTemp(currentItem)
-                    try await archive.add(currentItem)
+                    try await archive.add(currentItem, open: true)
```

Comment:
this line throws an error but we can't try different name because temp items were already moved